scripts/story-sync/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "story-sync",
+  "version": "1.0.0",
+  "private": true,
+  "description": "This script compares and syncs the stories between v3 and v4",
+  "license": "Apache-2.0",
+  "scripts": {
+    "compare": "ts-node ./compare.ts",
+    "sync": "ts-node ./sync.ts"
+  },
+  "dependencies": {
+    "react-scanner": "^1.0.0",
+    "rimraf": "^3.0.2"
+  },
+  "devDependencies": {
+    "@types/rimraf": "^3.0.2"
+  }
```

Comment:
It doesn't look like we're using these deps / devDeps

Reply:
Good point! That's a classic copy-and-paste error. 🤦